rockcraft/oci.py

```diff
@@ -191,7 +191,7 @@ def digest(source_image: str) -> bytes:
         :returns: The image digest bytes.
         """
         output = subprocess.check_output(
-            ["skopeo", "inspect", "--format", "{{.Digest}}", "-n", source_image],
```

Comment:
interesting; what's this about?

Reply:
Awww oops. That wasn't supposed to go in here, and I checked it TWICE but somehow still managed this.

The version of skopeo in 22.04 complains with the `-n` and wants just the image, and I was too lazy to grab the newer version from github.